FrontEnd/src/components/CompanyCard/CompanyCard.module.css

```diff
@@ -2,6 +2,7 @@
 
 .company-card {
   width: 360px;
+  height: 415px;
```

Comment:
Setting height usually causes many bugs with creating responsive sites. You may use min or max height depending on what you need

Reply:
Our site isn't responsible for now. And we don't have design to implement styles for responsible components. We are going to work on it after having the design

Reply:
Thank your for feedback. Done